docs/userGuide/syntax/variables.mbdf

```diff
@@ -56,7 +56,21 @@ Represents the root directory of the project. Used for specifying intra-site lin
 
 ##### Built-in Variable: `timestamp`
 
-<code>{<span></span>{ timestamp }}</code> is the time stamp (in UTC) that indicates when the page was generated.
+<code>{<span></span>{ timestamp }}</code> is the time stamp that indicates when the page was generated.
+
+You can set the time zone and locale in `site.json` like so - 
```

Comment:
Should this be moved to the site configuration page instead like the other options?
I'm personally not so familiar with the possible options for each field; Perhaps the format/standard being used here could be stated as well.

Reply:
Yes, you're right. It makes more sense for it to be in the configuration page. 
I'll link the possible options for both and explain the date and time format.

Reply:
On second thought, following the example of the `enableSearch` parameter, could we reference the `variables.mbdf` in `siteConfiguration.md` to explain the `timeZone` and `locale` parameters?

Reply:
Sounds good as well